pdf2image/pdf2image.py

```diff
@@ -105,17 +110,22 @@ def convert_from_path(pdf_path, dpi=200, output_folder=None, first_page=None, la
         else:
             images += parse_buffer_func(data)
 
+    for uid, proc in processes:
```

Comment:
What does this achieve?

Reply:
In Windows, sometimes the Poppler process are still active holding the handle for the input file. If try to remove the file immediately after pdf2image call, I fails.